plugins/modules/source_control/gitlab/gitlab_user.py

```diff
@@ -448,6 +490,7 @@ def main():
 
     gitlab_user = GitLabUser(module, gitlab_instance)
     user_exists = gitlab_user.existsUser(user_username)
+    user_is_active = gitlab_user.isActive(user_username)
```

Comment:
```suggestion
    if user_exists:
        user_is_active = gitlab_user.isActive(user_username)
    else:
        user_is_active = False
```
Or something like that

Reply:
This is the better way to go, I agree